app/models/miq_worker.rb

```diff
@@ -267,6 +267,8 @@ def self.start_worker(*params)
     w
   end
 
+  cache_with_timeout(:my_worker) { server_scope.find_by(:pid => Process.pid) }
```

Comment:
@kbrock no, I guess what I'm saying is it doesn't really make sense at the subclass level.  We only want this on MiqWorker and not be inherited.  I'm fine with it, just curious.  I wasn't sure if you tried it out.